lxd/main_init.go

```diff
@@ -128,6 +129,19 @@ func (c *cmdInit) Run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	// Check if the path to the cluster certificate is set
+	// If yes then read cluster certificate from file
+	if config.Cluster != nil && len(config.Cluster.ClusterCertificatePath) > 0 {
```

Comment:
We usually do `config.Cluster.ClusterCertificatePath != ""` instead of looking at the string length.